fe/fe-core/src/main/java/com/starrocks/warehouse/Warehouse.java

```diff
@@ -53,4 +54,6 @@ public void write(DataOutput out) throws IOException {
         String json = GsonUtils.GSON.toJson(this);
         Text.writeString(out, json);
     }
+
+    public abstract List<Long> getWorkerGroupIds();
```

Comment:
It is the clusterId from a warehouse abstraction. workerGroup Id is the starmgr implementation details.

name it to `getClusterIds()`

Reply:
The Cluster and Worker Group are both concept in epack, and there is no default clusterId.
The Worker Group has been used may places.

If we want hide the workerGroup concept in starmgr,  Can we open another PR to resolve it.

Reply:
The Cluster and Worker Group are both concept in epack, and there is no default clusterId.
The Worker Group has been used may places.

If we want hide the workerGroup concept in starmgr,  Can we open another PR to resolve it.

Reply:
forgot that the Cluster has been already removed. so be it. no need to worry about the cluster and workergroup now.